Add content from community doc PRs (916, 917, 918, 922) (#161)
rancher/harvester-product-docs: versions/v1.7/modules/en/pages/storage/longhorn-v2-data-engine.adoc

. Start the virtual machines.
+

> This step may take some time, depending on the amount of data that must be copied.

Reviewer: ```suggestion
This step may take some time, depending on the amount of data to copy.
```
